docs/source/compatibility-with-leapp-repository.md

```diff
@@ -0,0 +1,60 @@
+# Compatibility between leapp and leapp repositories
+
+There is hidden dragon in split of leapp framework and leapp repositories
+projects. The development of features and incompatible changes has different
+speed in both projects. As we want to release our projects using semantic
+versioning and in the same time, we do not want to release new version of
+the project after each merge of the new feature or incompatible change.
+We rather prefer releasing of new versions with changelogs and everything
+when we agree it's worthwhile.
+
+But during the time we have changes just in the upstream (master) branches,
+we need to ensure we are able to sync other leapp-repositories projects with
+the current devel changes. For these pruposes the `leapp-framework` capability
+is provided in framework (python[23]-leapp) rpms.
+
+## When and how change the capability
+
+The `leapp-framework` capability has to be changed in case of any change of
+provided API (e.g. change in leapp standard library) or functionality, that
+could affect actors in leapp repositories. That includes new features as well,
+as an actor could depend on that in future, and a leapp-repository package
+should be able to specify correct framework version that is needed.
+
+The `leapp-framework` capability uses just `X.Y` versioning:
+
+- In case of a change breaking previous functionality (e.g. changed a name of a function) provided by framework, bump `X` and set `Y` to 0.
```

Comment:
```suggestion
- In case of a change breaking previous functionality (e.g. changed a name of a function) provided by the framework, bump `X` and set `Y` to 0.
```